Text/HealthcareTA/HealthcareTA.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using AzureCognitiveSearch.PowerSkills.Common;
+using Azure.AI.TextAnalytics;
+using Azure;
+
+namespace AzureCognitiveSearch.PowerSkills.Text.HealthcareTA
+{
+    public static class HealthcareTA
+    {
+        private static readonly string healthcareApiEnvEndpoint = "HEALTHCARE_API_ENDPOINT";
+        private static readonly string healthcareApiEnvKey = "HEALTHCARE_API_KEY";
+        private static readonly int defaultTimeout = 230;
+        private static readonly int maxCharLength = 5000;
+
+        [FunctionName("HealthcareTA")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "post", Route = null)] HttpRequest req,
+            ILogger log,
+            ExecutionContext executionContext)
+        {
+            string skillName = executionContext.FunctionName;
+            IEnumerable<WebApiRequestRecord> requestRecords = WebApiSkillHelpers.GetRequestRecords(req);
+            if (requestRecords == null)
+            {
+                return new BadRequestObjectResult($"{skillName} - Invalid request record array.");
+            }
+
+            // Get Endpoint and access key from App Settings
+            string apiKey = Environment.GetEnvironmentVariable(healthcareApiEnvKey, EnvironmentVariableTarget.Process);
+            string apiEndpoint = Environment.GetEnvironmentVariable(healthcareApiEnvEndpoint, EnvironmentVariableTarget.Process);
```

Comment:
>string apiEndpoint = Environment.GetEnvironmentVariable(healthcareApiEnvEndpoint, EnvironmentVariableTarget.Process); [](http://example.com/codeflow?start=12&length=117)

Do we need the endpoint to be in an environment variable? It should be pretty standard and able to be hardcoded right? You might have copied this from the form recognizer example because that one does have a custom endpoint (since a customer has to pretain a form recognizer model) but this one shouldn't.

Reply:
I think that the TextAnalytics API *does* have a constant endpoint, but when allocating a resource for a text analytics account, the user also gets an endpoint within a unique subdomain. I figured since most people will be deploying this with the arm template we can use the endpoint with the subdomain, that way if the endpoint changes, it will automatically get updated.
I could also include a check where if there is not endpoint it defaults to the constant endpoint from the TextAnalytics API.

Reply:
Gotcha, yea let's just add a default to the constant one then

Reply:
This might also help with testing. Variable + default sounds good.

Reply:
Added a default for the endpoint